src/community/china.md

```diff
@@ -6,74 +6,171 @@ toc: true
 
 {% assign path = 'flutter_infra_release/releases/stable/windows/flutter_windows_3.3.0-stable.zip' -%}
 
-The Flutter community has made a Simplified Chinese version
-of the Flutter website available at
-[https://flutter.cn](https://flutter.cn),
-maintained by [The China Flutter User Group (CFUG)][].
+To speed the download and installation of Flutter in China,
+consider using a mirror site.
 
-If you'd like to install Flutter using an 
-[installation bundle]({{site.url}}/release/archive),
-you can replace the domain of the original URL with a trusted mirror
-to speed it up. For example:
+{{site.alert.important}}
+  Use mirror sites only if you _trust_ the provider.
+  The Flutter team cannot verify their reliability or security.
+{{site.alert.end}}
+
+## Use a Flutter mirror site
+
+The [China Flutter User Group][] (CFUG) maintains a Simplified Chinese
+Flutter website [https://flutter.cn](https://flutter.cn) and a mirror.
+Other mirrors can be found at the [end of this guide](#known-trusted-community-run-mirror-sites).
+
+### Configure your machine to use a mirror site
+
+To install or use Flutter in China,
+use a trustworthy Flutter mirror site.
+This requires setting two environment variables on your machine.
+
+_All examples that follow presume that you are using the CFUG mirror._
+
+To set your machine to use a mirror site on macOS or Linux:
```

Comment:
Consider adding the Windows part too. For Windows users, the command with PowerShell would be:
```powershell
$env:PUB_HOSTED_URL="https://pub.flutter-io.cn"
```

Reply:
![image](https://github.com/flutter/website/assets/15884415/204d9df1-fa2b-4084-906f-b3b798857c35)